README.md

```diff
@@ -973,6 +972,61 @@ Learn more about class-transformer and how to handle more complex object constru
 This behaviour is enabled by default.
 If you want to disable it simply pass `useClassTransformer: false` to createExpressServer method.
 
+## Auto validating action params
+
+Sometimes parsing a json object into instance of some class is not enough. 
+E.g. `class-transformer` doesn't check whether the property's types are correct, so you can get runtime error if you rely on TypeScript type safe. Also you may want to validate the object to check e.g. whether the password string is long enough or entered e-mail is correct.
+
+It can be done easily thanks to integration with [class-transformer-validator][7] which internally depends on [class-validator][6]. So first of all, you have to install the modules:

```

Comment:
> class-transformer-validator isn't tightly coupled with routing-controllers, it can work with anything, eg. with pure express.js:

yes Ive got this.


I hate dependencies. Dependencies makes software more complex and harder to maintain. Especially small dependencies. When there is no real need to have extra dependency and maintainability cost for such a small functionality that can take just few lines of code there.